python/aswfdocker/builder.py

```diff
@@ -113,8 +140,14 @@ def make_bake_dict(self) -> typing.Dict[str, dict]:
         root["group"] = {"default": {"targets": list(root["target"].keys())}}
         return root
 
-    def make_bake_jsonfile(self) -> typing.Optional[str]:
-        d = self.make_bake_dict()
+    def make_bake_jsonfile(
+        self,
+        keep_source: bool,
```

Comment:
You could make this function keyword only to avoid confusion on what does the Boolean does. You can do it by putting a * as argument before keep_source. This with default values makes it easier to use and less error prone